### ISSUE TITLE
Save As icon

### ISSUE THREAD

**Issue by aforlani** [Templarian/MaterialDesign#1134]
Please, add a Save As icon.

Thanks in advance,
Andrea

**Comment by Templarian** [comment]
Standard icon for this would probably be a disk + pencil I assume. Shrinking the pencil always looks odd, but might be the best option.

3 dots is already used for settings icons, so we can't use that.

**Comment by OlsonDev** [comment]
That or two stacked floppies is pretty common:

![image](https://cloud.githubusercontent.com/assets/308072/14293445/30bca8e6-fb32-11e5-9214-c2d1da394f92.png)

**Comment by Templarian** [comment]
@OlsonDev That exists, but is more `save all` isn't it?

**Comment by OlsonDev** [comment]
Yeah, you're right. I'm going through my installed apps and most don't even have an icon for `Save as` but I do see a few with the pencil. :-\

**Comment by aforlani** [comment]
Thank you for your replies :) Tipically the icon for "Save As", as Austin says, is the one with Disk + Pencil. Yes the "Save All" one already exists and is perfect.

**Comment by jfoxtail** [comment]
![saveas](https://cloud.githubusercontent.com/assets/8354098/14393470/0bc9a2fe-fd7c-11e5-841c-e6529ce954b0.png)
This could be another option for you.

**Comment by CoDEmanX** [comment]
![save_as](https://cloud.githubusercontent.com/assets/1309249/14400694/8a322640-fdfc-11e5-968d-961436747591.png) original icon moved 1 pixel up
![save_all](https://cloud.githubusercontent.com/assets/1309249/14400695/8a32b114-fdfc-11e5-8c70-5107479d7e7b.png) not sure about this one

**Comment by OlsonDev** [comment]
@Templarian already suggested those ideas don't fit the theme of other icons:

> 3 dots is already used for settings icons, so we can't use that.

**Comment by CoDEmanX** [comment]
3 dots stand for "more" in my opinion. Bluetooth plus ... = more Bluetooth (here: options). 

It is to be expected, that some kind of dialog is opened if an icon with three dots is clicked. In case of the save-as action, **there is** a dialog - the one that lets you choose the target folder and name - as opposed to the save action, which only opens a dialog if it's a new file.

**Comment by OlsonDev** [comment]
I mean, it makes sense to me to use the "ellipse" for a dialog-invoking action -- just wanted what he said to be addressed in case anyone had better ideas/missed it. :smile:

That said, I don't have any better ideas. :-\

**Comment by Templarian** [comment]
[![content-save-settings](https://materialdesignicons.com/icon/content-save-settings)](https://materialdesignicons.com/icon/content-save-settings)

I think this will work. I'm keeping the naming `-settings` for consistency, but you can still definitely use it for save as (in context it would fit I think).

**Comment by CoDEmanX** [comment]
There's also "save copy as..." in many applications, but I can't think of another icon.

**Comment by aforlani** [comment]
Sorry if I could not read this thread during the weekend. Thanks for your work. Now I was considering the icon with Disk + three dots. I think it's perfect for Save or Export options/ preferences (ex. compression type, ratio, default save/export path....) but I'm not sure it's the best option for Save As interaction where, typically, a file dialog opens requesting only the filename. Don't you think that the traditional Disk + Pencil icon is the best solution? The one that users match to Save As action in a more conventional way?

**Comment by CoDEmanX** [comment]
As Templarian mentioned, it is the size restrictions that make floppy + pencil particularly hard to fit in the 24 px grid. We should give it a try nonetheless.

What about save-multiple BTW? Save as sort of creates a duplicate of an existing file, so there a two files afterwards, hence multiple.

**Comment by aforlani** [comment]
I understand the tech difficulty but I'm concerned about the user expectation. He knows that Disk + pencil is SaveAs.  Save-multiple, for me, is just fine for another well known and specific action: SaveAll. In many softwares there are both SaveAs and SaveAll buttons and it would be difficult to represent them using only one icon.

**Comment by goyney** [comment]
We use the smaller pencil in several icons now. Here are the two icons discussed in this issue:
![image](https://user-images.githubusercontent.com/1015040/48818668-3c377480-ed1a-11e8-9e64-732fc08fe1cb.png)![image](https://user-images.githubusercontent.com/1015040/48818677-43f71900-ed1a-11e8-9341-ff20caf025df.png)

[content-save-edit.zip](https://github.com/Templarian/MaterialDesign/files/2602609/content-save-edit.zip)
